app/models/assessment.rb

```diff
@@ -28,6 +29,8 @@ def self.seed_somerville_assessments
     Assessment.create!([
       { family: "MCAS", subject: "Mathematics" },
       { family: "MCAS", subject: "ELA" },
+      { family: "MCAS", subject: "Mathematics", flavor: "Next Generation MCAS" },
+      { family: "MCAS", subject: "ELA", flavor: "Next Generation MCAS" },
```

Comment:
Adding `flavor` seems like it adds complexity to the whole data model for this particular assessment.  I'm wondering how it compares to using the `family` field and calling these "Next Generation MCAS" - are there benefits to adding another field for this particular assessment?

Reply:
@kevinrobinson Yeah, the benefit I have in mind is keeping the code path simple for all of these attributes on the student model:

```
+  most_recent_mcas_math_growth
+  most_recent_mcas_ela_growth
+  most_recent_mcas_math_performance
+  most_recent_mcas_ela_performance
+  most_recent_mcas_math_scaled
+  most_recent_mcas_ela_scaled
```

For all these attributes, we don't care if the latest MCAS result is the old kind or the new kind — we're just surfacing a number or a string, and we don't necessarily need to add a lot of context around it (for example, in the homeroom view).

If we make a new family called "Next Generation MCAS" we'll have to update the code paths for all of these attributes and merge together Next Generation MCAS results with old-style MCAS results.

Reply:
Yeah, that makes sense!  I was wondering if we wanted to render the two kinds of assessments the same way if they were on different scales and had different performance levels.  In a table like on the homeroom view, would we want those two kinds of scores to look the same and be in the same column?

My assumption is that it might be important to add that context, so that when you ask how a student did on MCAS, you probably need to know whether it was MCAS or MCAS 2.0 and how it was scaled in order to understand what that number means.

Reply:
Let's ask @snoopyuri what his assumption is? Since the two ranges don't overlap at all (200 to 270 vs 400 to 600) my assumption is that educators won't need an explicit label to know if they're looking at a 1.0 or a 2.0 score.